client/lib/analytics/index.js

```diff
@@ -95,6 +95,14 @@ const analytics = {
 		_superProps = superProps;
 	},
 
+	checkDoNotTrack: function() {
+		if ( navigator.doNotTrack ) {
+			return false;
+		}
+
+		return true;
+	},
```

Comment:
`checkDoNotTrack` is a pretty confusing name - it returns true if the user has not configured the brower with 'do not track' (so allows tracking)? :)

May something like `userAllowsTracking`?